### ISSUE TITLE
make dep failing on sourceforge dependency

### ISSUE THREAD

**Issue by MarcBoule** [VCVRack/VCV-Recorder#9]
My `make dep` is failing while fetching some sourceforge packages. I'll put an excerpt of my terminal below, perhaps you might see something useful (sorry for the french, but the penultimate line says that 20 redirections occured, which is beyond the acceptable limit). I've been having this error in the previous days as well, but figured it was because the repo was not finished. 

(Win10, commit https://github.com/VCVRack/VCV-Recorder/commit/4255b048fdbc416f17ca0f00a1c14340cd34d162)

(terminal log below was from a second invocation of `make dep`, so some things were already done)

````
$ make dep
cd dep && wget -c "https://sourceforge.net/projects/lame/files/lame/3.100/lame-3.100.tar.gz"
--2019-04-17 12:03:18--  https://sourceforge.net/projects/lame/files/lame/3.100/lame-3.100.tar.gz
Résolution de sourceforge.net (sourceforge.net)… 216.105.38.13
Connexion à sourceforge.net (sourceforge.net)|216.105.38.13|:443… connecté.
requête HTTP transmise, en attente de la réponse… 301 Moved Permanently
Emplacement : https://sourceforge.net/projects/lame/files/lame/3.100/lame-3.100.tar.gz/ [suivant]
--2019-04-17 12:03:19--  https://sourceforge.net/projects/lame/files/lame/3.100/lame-3.100.tar.gz/
Connexion à sourceforge.net (sourceforge.net)|216.105.38.13|:443… connecté.
requête HTTP transmise, en attente de la réponse… 302 Found
Emplacement : https://sourceforge.net/projects/lame/files/lame/3.100/lame-3.100.tar.gz/download [suivant]
--2019-04-17 12:03:19--  https://sourceforge.net/projects/lame/files/lame/3.100/lame-3.100.tar.gz/download
Connexion à sourceforge.net (sourceforge.net)|216.105.38.13|:443… connecté.
requête HTTP transmise, en attente de la réponse… 302 Found
Emplacement : https://downloads.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz?r=&ts=1555516999&use_mirror=ayera [suivant]
--2019-04-17 12:03:20--  https://downloads.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz?r=&ts=1555516999&use_mirror=ayera
Résolution de downloads.sourceforge.net (downloads.sourceforge.net)… 216.105.38.13
Connexion à downloads.sourceforge.net (downloads.sourceforge.net)|216.105.38.13|:443… connecté.
requête HTTP transmise, en attente de la réponse… 302 Found
Emplacement : https://ayera.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz [suivant]
--2019-04-17 12:03:20--  https://ayera.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz
Résolution de ayera.dl.sourceforge.net (ayera.dl.sourceforge.net)… 172.110.242.68, 2607:fa38:0:fe:6563:4fc1:ac91:ea86
Connexion à ayera.dl.sourceforge.net (ayera.dl.sourceforge.net)|172.110.242.68|:443… connecté.
requête HTTP transmise, en attente de la réponse… 302 Moved Temporarily
Emplacement : http://downloads.sourceforge.net/mirrorproblem?failedmirror=ayera.dl.sourceforge.net [suivant]
--2019-04-17 12:03:21--  http://downloads.sourceforge.net/mirrorproblem?failedmirror=ayera.dl.sourceforge.net
Connexion à downloads.sourceforge.net (downloads.sourceforge.net)|216.105.38.13|:80… connecté.
requête HTTP transmise, en attente de la réponse… 302 Found
Emplacement : https://iweb.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz [suivant]
--2019-04-17 12:03:21--  https://iweb.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz
Résolution de iweb.dl.sourceforge.net (iweb.dl.sourceforge.net)… 192.175.120.182, 2607:f748:10:12::5f:2
Connexion à iweb.dl.sourceforge.net (iweb.dl.sourceforge.net)|192.175.120.182|:443… connecté.
requête HTTP transmise, en attente de la réponse… 302 Found
Emplacement : http://downloads.sourceforge.net/mirrorproblem?failedmirror=iweb.dl.sourceforge.net [suivant]
--2019-04-17 12:03:21--  http://downloads.sourceforge.net/mirrorproblem?failedmirror=iweb.dl.sourceforge.net
Réutilisation de la connexion existante à downloads.sourceforge.net:80.
requête HTTP transmise, en attente de la réponse… 302 Found
Emplacement : https://superb-dca2.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz [suivant]
--2019-04-17 12:03:21--  https://superb-dca2.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz
Résolution de superb-dca2.dl.sourceforge.net (superb-dca2.dl.sourceforge.net)… 209.61.193.20
Connexion à superb-dca2.dl.sourceforge.net (superb-dca2.dl.sourceforge.net)|209.61.193.20|:443… connecté.
requête HTTP transmise, en attente de la réponse… 302 Moved Temporarily
Emplacement : http://downloads.sourceforge.net/mirrorproblem?failedmirror=superb-dca2.dl.sourceforge.net [suivant]
--2019-04-17 12:03:21--  http://downloads.sourceforge.net/mirrorproblem?failedmirror=superb-dca2.dl.sourceforge.net
Réutilisation de la connexion existante à downloads.sourceforge.net:80.
requête HTTP transmise, en attente de la réponse… 302 Found
Emplacement : https://astuteinternet.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz [suivant]
--2019-04-17 12:03:21--  https://astuteinternet.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz
Résolution de astuteinternet.dl.sourceforge.net (astuteinternet.dl.sourceforge.net)… 162.213.157.36
Connexion à astuteinternet.dl.sourceforge.net (astuteinternet.dl.sourceforge.net)|162.213.157.36|:443… connecté.
requête HTTP transmise, en attente de la réponse… 302 Moved Temporarily
Emplacement : http://downloads.sourceforge.net/mirrorproblem?failedmirror=astuteinternet.dl.sourceforge.net [suivant]
--2019-04-17 12:03:22--  http://downloads.sourceforge.net/mirrorproblem?failedmirror=astuteinternet.dl.sourceforge.net
Réutilisation de la connexion existante à downloads.sourceforge.net:80.
requête HTTP transmise, en attente de la réponse… 302 Found
Emplacement : https://phoenixnap.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz [suivant]
--2019-04-17 12:03:22--  https://phoenixnap.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz
Résolution de phoenixnap.dl.sourceforge.net (phoenixnap.dl.sourceforge.net)… 184.164.141.26
Connexion à phoenixnap.dl.sourceforge.net (phoenixnap.dl.sourceforge.net)|184.164.141.26|:443… connecté.
requête HTTP transmise, en attente de la réponse… 302 Moved Temporarily
Emplacement : http://downloads.sourceforge.net/mirrorproblem?failedmirror=phoenixnap.dl.sourceforge.net [suivant]
--2019-04-17 12:03:22--  http://downloads.sourceforge.net/mirrorproblem?failedmirror=phoenixnap.dl.sourceforge.net
Réutilisation de la connexion existante à downloads.sourceforge.net:80.
requête HTTP transmise, en attente de la réponse… 302 Found
Emplacement : https://managedway.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz [suivant]
--2019-04-17 12:03:22--  https://managedway.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz
Résolution de managedway.dl.sourceforge.net (managedway.dl.sourceforge.net)… 208.79.213.6
Connexion à managedway.dl.sourceforge.net (managedway.dl.sourceforge.net)|208.79.213.6|:443… connecté.
requête HTTP transmise, en attente de la réponse… 302 Moved Temporarily
Emplacement : http://downloads.sourceforge.net/mirrorproblem?failedmirror=managedway.dl.sourceforge.net [suivant]
--2019-04-17 12:03:22--  http://downloads.sourceforge.net/mirrorproblem?failedmirror=managedway.dl.sourceforge.net
Réutilisation de la connexion existante à downloads.sourceforge.net:80.
requête HTTP transmise, en attente de la réponse… 302 Found
Emplacement : https://svwh.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz [suivant]
--2019-04-17 12:03:22--  https://svwh.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz
Résolution de svwh.dl.sourceforge.net (svwh.dl.sourceforge.net)… 72.5.72.15, 2606:c680:0:b:3830:34ff:fe66:6663
Connexion à svwh.dl.sourceforge.net (svwh.dl.sourceforge.net)|72.5.72.15|:443… connecté.
requête HTTP transmise, en attente de la réponse… 302 Moved Temporarily
Emplacement : http://downloads.sourceforge.net/mirrorproblem?failedmirror=svwh.dl.sourceforge.net [suivant]
--2019-04-17 12:03:23--  http://downloads.sourceforge.net/mirrorproblem?failedmirror=svwh.dl.sourceforge.net
Réutilisation de la connexion existante à downloads.sourceforge.net:80.
requête HTTP transmise, en attente de la réponse… 302 Found
Emplacement : https://versaweb.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz [suivant]
--2019-04-17 12:03:23--  https://versaweb.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz
Résolution de versaweb.dl.sourceforge.net (versaweb.dl.sourceforge.net)… 162.251.232.173
Connexion à versaweb.dl.sourceforge.net (versaweb.dl.sourceforge.net)|162.251.232.173|:443… connecté.
requête HTTP transmise, en attente de la réponse… 302 Moved Temporarily
Emplacement : http://downloads.sourceforge.net/mirrorproblem?failedmirror=versaweb.dl.sourceforge.net [suivant]
--2019-04-17 12:03:23--  http://downloads.sourceforge.net/mirrorproblem?failedmirror=versaweb.dl.sourceforge.net
Réutilisation de la connexion existante à downloads.sourceforge.net:80.
requête HTTP transmise, en attente de la réponse… 302 Found
Emplacement : https://cfhcable.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz [suivant]
--2019-04-17 12:03:23--  https://cfhcable.dl.sourceforge.net/project/lame/lame/3.100/lame-3.100.tar.gz
Résolution de cfhcable.dl.sourceforge.net (cfhcable.dl.sourceforge.net)… 146.71.73.6
Connexion à cfhcable.dl.sourceforge.net (cfhcable.dl.sourceforge.net)|146.71.73.6|:443… connecté.
requête HTTP transmise, en attente de la réponse… 302 Moved Temporarily
Emplacement : http://downloads.sourceforge.net/mirrorproblem?failedmirror=cfhcable.dl.sourceforge.net [suivant]
20 redirections dépassant la limite permise.
make: *** [Makefile:43: dep/lib/libmp3lame.a] Error 8
```

**Comment by antoniotuzzi** [comment]
now, same for me (I was able the first time)

**Comment by MarcBoule** [comment]
Tried it again, and now I'm getting your error from issue #10 Antonio
```
libtool: link:  gcc -shared .libs/libmp3lame-0.dll.def  .libs/VbrTag.o .libs/bitstream.o .libs/encoder.o .libs/fft.o .libs/gain_analysis.o .libs/id3tag.o .libs/lame.o .libs/newmdct.o .libs/presets.o .libs/psymodel.o .libs/quantize.o .libs/quantize_pvt.o .libs/reservoir.o .libs/set_get.o .libs/tables.o .libs/takehiro.o .libs/util.o .libs/vbrquantize.o .libs/version.o .libs/mpglib_interface.o  -Wl,--whole-archive ../libmp3lame/vector/.libs/liblamevectorroutines.a ../mpglib/.libs/libmpgdecoder.a -Wl,--no-whole-archive   -g -O3 -march=nocona   -o .libs/libmp3lame-0.dll -Wl,--enable-auto-image-base -Xlinker --out-implib -Xlinker .libs/libmp3lame.dll.a
C:/msys64/mingw64/bin/../lib/gcc/x86_64-w64-mingw32/8.3.0/../../../../x86_64-w64-mingw32/bin/ld.exe: Cannot export lame_init_old: symbol not defined
collect2.exe: error: ld returned 1 exit status
make[3]: *** [Makefile:550: libmp3lame.la] Error 1
make[3] : on quitte le répertoire « /home/mboule/Rack/rdev/Rack/plugins/VCV-Recorder/dep/lame-3.100/libmp3lame »
make[2]: *** [Makefile:613: install-recursive] Error 1
make[2] : on quitte le répertoire « /home/mboule/Rack/rdev/Rack/plugins/VCV-Recorder/dep/lame-3.100/libmp3lame »
make[1]: *** [Makefile:452: install-recursive] Error 1
make[1] : on quitte le répertoire « /home/mboule/Rack/rdev/Rack/plugins/VCV-Recorder/dep/lame-3.100 »
make: *** [Makefile:46: dep/lib/libmp3lame.a] Error 2
```

**Comment by antoniotuzzi** [comment]
check my suggestion:
https://github.com/VCVRack/VCV-Recorder/issues/10#issuecomment-484159580

**Comment by Coirt** [comment]
Did not work for me anyway.

**Comment by AndrewBelt** [comment]
The issue is because SourceForge mirrors don't understand the HTTP header that `wget -c` sends.

Fixed in 87f6e5588b49e2043840b7c974466d5b96571b51 by telling wget to skip downloading if the file exists. Pull the latest Rack commits.
